exasol/slc_ci/cli/options/branch_options.py

```diff
@@ -0,0 +1,17 @@
+import click
+
+branch_option = click.option(
+    "--branch-name",
+    type=str,
+    required=True,
+    help="In case of a PR, the source branch of the PR.",
+)
+
+base_branch_option = click.option(
+    "--base-branch-name",
```

Comment:
It is usually a base-ref and not a branch. A ref can be anywhere in the history, it can be a commit hash, tag or branch

Reply:
Fixed